partition_snapshot_row_cursor.hh

```diff
@@ -641,7 +641,7 @@ public:
     friend std::ostream& operator<<(std::ostream& out, const partition_snapshot_row_cursor& cur) {
         out << "{cursor: position=" << cur._position << ", cont=" << cur.continuous() << ", ";
         if (cur._reversed) {
-            out << "rev=1, ";
+            out << "reversed, ";
```

Comment:
I think this can be just push to the previous patch when it was introduced.

Reply:
right